src/main/java/software/amazon/event/kafkaconnector/EventBridgeSinkConfig.java

```diff
@@ -127,6 +124,12 @@ private static void addParams(final ConfigDef configDef) {
     configDef.define(AWS_EVENTBUS_ARN_CONFIG, Type.STRING, Importance.HIGH, AWS_EVENTBUS_ARN_DOC);
     configDef.define(
         AWS_ENDPOINT_URI_CONFIG, Type.STRING, "", Importance.MEDIUM, AWS_ENDPOINT_URI_DOC);
+    configDef.define(
+        AWS_EVENTBUS_GLOBAL_ENDPOINT_ID_CONFIG,
+        Type.STRING,
+        "",
```

Comment:
is this the default value? if yes, then would `endpointId == null` ever return true?

Reply:
Yes, empty string is default. NULL check is just a habit of me - I've seen frameworks changing behavior from empty string to NULL causing NPEs ,)

Reply:
Btw: it's an `OR` condition so never being `null` also doesn't affect the logic